src/android/src/main/java/io/radar/cordova/RadarCordovaPlugin.java

```diff
@@ -389,6 +446,21 @@ public void offError(final JSONArray args, final CallbackContext callbackContext
         RadarCordovaPlugin.errorCallbackContext = null;
     }
 
+    public void startTrip(final JSONArray args, final CallbackContext callbackContext) throws JSONException {
+        final JSONObject optionsObj = args.getJSONObject(0);
+
+        RadarTripOptions options = RadarTripOptions.fromJson(optionsObj);
+        Radar.startTrip(options);
+
+        callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK));
+    }
+
+    public void stopTrip(final JSONArray args, final CallbackContext callbackContext) throws JSONException {
+        Radar.stopTrip();
+
+        callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK));
```

Comment:
these two functions look like they're sending back `STATUS.OK` regardless of what our API reports - does Cordova have a similar construct to `try / catch`?